### DISCUSSION TITLE
Sending effect description to chat using token action HUD

### DISCUSSION THREAD

**Discussion by leborat** [DFreds/dfreds-convenient-effects#336] [General]
is there a way to send the effect description from Dfreds convenient effects using the token action hud effects?
as in, send this effect message: 
![image](https://github.com/user-attachments/assets/b7637bce-ff13-47c1-94b9-89a25d767b8d)

using these buttons 
![image](https://github.com/user-attachments/assets/0b069708-7465-42eb-b3c6-1907c03e4f9e)

**Comment by DFreds** [comment]
This used to be a feature of this module, but I removed it due to it being outside the scope of the intention of this module. I'm going to create a new module that explicitly does this
